<a href="https://colab.research.google.com/github/pingao2019/lab_data/blob/master/dataset_bridge_to_property1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
import pandas as pd
sites=pd.read_excel("/content/B2P-RwandaSiteAssessment.xlsx")
sites.head()

Country  ... Bridge Opportunity: Opportunity ID
0  Rwanda  ...                    006f100000d1fk1
1  Rwanda  ...                    006f100000d1fjw
2  Rwanda  ...                    006f100000a86I3
3  Rwanda  ...                    006f100000cPpL8
4  Rwanda  ...                    006f100000eescb

[5 rows x 27 columns]

In [2]:
sites.shape

(1472, 27)

In [3]:
sites.columns.to_list()

['Country',
 'Province',
 'District',
 'Sector',
 'Cell',
 'Bridge Site Name',
 'Project Stage',
 'Project Sub-Stage',
 'Project Code',
 'Bridge Type',
 ' Span (m)',
 ' GPS (Latitude)',
 'GPS (Longitude)',
 'Individuals Directly Served',
 'Form: Form Name',
 'Community Served 1',
 'Community Served 2',
 'Community Served 3',
 'Community Served 4',
 'Community Served 5',
 'Community Served 6',
 'Community Served 7',
 'Community Served 8',
 'Community Served 9',
 'Community Served 10',
 'CaseSafeID Form',
 'Bridge Opportunity: Opportunity ID']

Summary of bridge sites data: 27 columns, 1472 samples(rows). Gives informations about : "'Country', 'Province', 'District', 'Sector', 'Cell', 'Bridge Site Name', 'Project Stage', 'Project Sub-Stage', 'Project Code', 'Bridge Type', ' Span (m)', ' GPS (Latitude)', 'GPS (Longitude)', 'Individuals Directly Served', 'Form: Form Name', 'Community Served 1', 'Community Served 2', 'Community Served 3', 'Community Served 4', 'Community Served 5', 'Community Served 6', 'Community Served 7', 'Community Served 8', 'Community Served 9', 'Community Served 10', 'CaseSafeID Form', 'Bridge Opportunity: Opportunity ID'"

In [5]:
import pandas as pd
villages=pd.read_excel("/content/Rwanda-AdministrativeVillage .xlsx")
villages.head()

Prov_ID   Province  Dist_ID  District  ...   Vill_ID    Village  Status   FID
0        2  Amajyepfo       22  Gisagara  ...  22050101    Agahehe   Rural  1842
1        2  Amajyepfo       22  Gisagara  ...  22050102   Kabacuzi   Rural  1846
2        2  Amajyepfo       22  Gisagara  ...  22050103   Kamutozo   Rural  1854
3        2  Amajyepfo       22  Gisagara  ...  22050104    Kamweko   Rural  1866
4        2  Amajyepfo       22  Gisagara  ...  22050105  Nyamabuye   Rural  1873

[5 rows x 12 columns]

In [6]:
villages.shape

(14816, 12)

In [7]:
villages.columns.to_list()

['Prov_ID',
 'Province',
 'Dist_ID',
 'District',
 'Sect_ID',
 'Sector',
 'Cell_ID',
 'Cell',
 'Vill_ID',
 'Village',
 'Status',
 'FID']